<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-17T16:13:51 IST" name="Default suite" finished-at="2022-11-17T16:14:15 IST" duration-ms="23481">
    <groups>
    </groups>
    <test started-at="2022-11-17T16:13:51 IST" name="Default test" finished-at="2022-11-17T16:14:15 IST" duration-ms="23481">
      <class name="waletpomclass.Testcaseqrcode">
        <test-method signature="scanningqrcode()[pri:0, instance:waletpomclass.Testcaseqrcode@130161f7]" started-at="2022-11-17T16:13:51 IST" name="scanningqrcode" finished-at="2022-11-17T16:14:15 IST" duration-ms="23449" status="FAIL">
          <exception class="java.net.MalformedURLException">
            <message>
              <![CDATA[unknown protocol: data]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.net.MalformedURLException: unknown protocol: data
at java.base/java.net.URL.<init>(URL.java:681)
at java.base/java.net.URL.<init>(URL.java:569)
at java.base/java.net.URL.<init>(URL.java:516)
at waletpomclass.QRcode.scanqrcode(QRcode.java:40)
at waletpomclass.Testcaseqrcode.scanningqrcode(Testcaseqrcode.java:26)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.net.MalformedURLException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scanningqrcode -->
      </class> <!-- waletpomclass.Testcaseqrcode -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
